openapi.yaml

```
@@ -3030,6 +3030,12 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/fileWithDocuments'
+        keys:
+          type: object
+          properties:
+            custom:
+              type: object
+              description: key-value pair of custom variables which were set for a run. 
```

```suggestion
              description: key-value pair of custom keys which were set for a run. 
```

the language here should match the language in our docs (ie `custom keys`)

also, should we link to the docs page explaining how custom keys are used? ie https://docs.instabase.com/automate/custom-functions#adding-custom-and-secret-keys-in-custom-functions